src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ResourceQueriesQueryString.feature.disabled

```diff
@@ -0,0 +1,63 @@
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by valid date
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate=2024-05-15"
+             Then it should respond with 200
+              And response body is
+              """
+              {"id":"f0aad8e209b64134b4c056fa9f49eeff","schoolReference":{"schoolId":255901001,"link":{"rel":"School","href":"/ed-fi/schools/56f9effc3c7f44f496fd8d6783dd9360"}},"weekIdentifier":"Week One","beginDate":"2024-05-15","endDate":"2024-05-15","totalInstructionalDays":2,"_etag":"5250200021872954522","_lastModifiedDate":"2024-05-15T20:05:44.5566618Z"}
+              """
+
+        @ignore
+        Scenario: Ensure clients can't GET information when querying by invalid date
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate=024-04-09"
+             Then it should respond with 400
+              And response body is
+                  """
+                  {"detail":"Data validation failed. See 'validationErrors' for details.","type":"urn:ed-fi:api:bad-request:data","title":"Data Validation Failed","status":400,"correlationId":null,"validationErrors":{"$.beginDate":["The value '024-04-09' is not valid for BeginDate."]}}
+                  """
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by date out of specified dates
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate=1970-04-09"
+             Then it should respond with 200
+              And response body is
+              """
+              {[]}
+              """
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by begin date and end date that are not part of the data
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate=2024-05-20&endDate=2024-05-20"
+             Then it should respond with 200
+              And response body is
+              """
+              {[]}
+              """
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by string identifier
+             When sending a GET request to "/ed-fi/academicWeeks?weekIdentifier=Week+One"
+             Then it should respond with 200
+              And response body is
+                  """
+                  {"id":"f0aad8e209b64134b4c056fa9f49eeff","schoolReference":{"schoolId":255901001,"link":{"rel":"School","href":"/ed-fi/schools/56f9effc3c7f44f496fd8d6783dd9360"}},"weekIdentifier":"Week One","beginDate":"2024-05-15","endDate":"2024-05-15","totalInstructionalDays":2,"_etag":"5250200021872954522","_lastModifiedDate":"2024-05-15T20:05:44.5566618Z"}
+                  """
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by integer identifier
+             When sending a GET request to "/ed-fi/academicWeeks?totalInstructionalDays=2"
+             Then it should respond with 200
+              And response body is
```

Comment:
This should be `And the response body is`

Reply:
Done